app/controllers/campaigns_controller.rb

```diff
@@ -0,0 +1,40 @@
+class CampaignsController < ApplicationController
+	def index
+		@campaigns = Campaign.all
+	end
+
+	def create
+		@campaign = Campaign.new(campaign_params)
+		if @campaign.save
+			redirect_to @campaign
+		else
+			render 'new'
+		end
+	end
+
+	def new
+		@campaign = Campaign.new
+	end
+
+	def show
+		@campaign = Campaign.find(params[:id])
+	end
+
+	def edit
+		@campaign = Campaign.find(params[:id])
+	end
+
+	def update
+		@campaign = Campaign.find(params[:id])
+		if @campaign.update(campaign_params)
+			redirect_to @campaign
+		else
+			render "edit"
+		end
+	end
+
+	private
```

Comment:
@korey-enright Not sure I understand this. Won't I have to invoke that method inside the show edit and update actions either way to set that variable since that variable is scoped to the `set_campaign` method? This feels like trading a one like object lookup for a method that's a one line object lookup. Or am I missing something here?